flake.nix

```diff
@@ -26,7 +26,13 @@
           pname = manifest.package.name;
           version = manifest.workspace.package.version;
           src = gitignoreSource ./.;
-          cargoLock.lockFile = ./Cargo.lock;
+          cargoLock = {
+            lockFile = ./Cargo.lock;
+            # Temporary fix until https://github.com/mattwparas/steel/issues/192 is fixed upstream
+            outputHashes = {
+              "lasso-0.7.2" = "sha256-ibpHfge3nEtwLNghKEQT7ZpTe5kgDf8hbBb9qYHyHcQ=";
```

Comment:
why is this necessary?

Reply:
I'm not sure exactly why but the `buildRustPackage` function when using a cargo file with a dependency that points to Git it needs this output hash defined. 

Without this hash if I run `nix run` I get:

```nix
       error: No hash was found while vendoring the git dependency lasso-0.7.2. You can add
       a hash through the `outputHashes` argument of `importCargoLock`:

       outputHashes = {
         "lasso-0.7.2" = "<hash>";
       };

       If you use `buildRustPackage`, you can add this attribute to the `cargoLock`
       attribute set.
```